### PR TITLE
APIs readme

### PR DESCRIPTION
**Pull request by tpluscode**
Some notes on the back end project structure etc

### CODE REVIEW

File: apis/readme.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    │   ├── domain/
+    │   └── handlers/
+    └── test/
+```
+
+### `bootstrap/`
+
+Contains static resources which are inserted into the triple store at application start to ensure it is functional. This is because every resource exposed by the API requires its representation stored in the database. This includes the API root (entrypoint) as well as "virtual" resources such as [collections][collection] and write-only resources.
+
+[collection]: http://www.hydra-cg.com/spec/latest/core/#collections
```
Comment (lucafurrer) [inline apis/readme.md:31]: Hi
A markdown linter hinted once to me, that bare urls are not that well supported in Markdown
https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md. 

```suggestion
[collection]: <http://www.hydra-cg.com/spec/latest/core/#collections>
```

Comment (vhf) [reply apis/readme.md:31]: It's not a bare url, it's a link reference, see it that way: `[title](something)`, something is the URL of the link, not a bare url in a text block. ;)

**Review by vhf — APPROVED** [review]

File: apis/readme.md (hunk 1 above)
Comment (lucafurrer) [reply apis/readme.md:31]: Missed that, sorry

**Review by vhf — APPROVED** [review]

**Comment by martinmaillard** [comment]
I like this PR :)

**Pull request merged**